doc/netplan.md

```diff
@@ -242,8 +242,31 @@ Virtual devices
     configured and DHCP is disabled, the interface may still be brought online,
     but will not be addressable from the network.
 
+    In addition to the addresses themselves one can specify configuration
+    parameters as mappings. Current supported options are:
+
+    ``lifetime`` (scalar) - since **0.100**
+    :    Default: ``forever``. This can be ``forever`` or ``0`` and corresponds
+         to the ``PreferredLifetime`` option in ``systemd-networkd``'s Address
+         section. Currently supported on the ``networkd`` backend only.
+
+    ``label`` (scalar) - since **0.100**
+    :    An IP address label, corresponding to the ``Label`` option in
+         ``systemd-networkd``. Currently supported on the ``networkd`` backend
```

Comment:
Thanks.  I think it would be better to document this in terms of the 'ip' command rather than in terms of 'systemd-networkd', since the former is the least common denominator.

Reply:
Ok, one more try  :)
Please let me know if you'd like more details on it!

Reply:
lgtm, thank you